plos_connector/plos_unittests.py

```diff
@@ -0,0 +1,96 @@
+import requests
+import requests_mock
+import unittest
+from main import *
```

Comment:
which main is this?

Reply:
main.py - the only main under this directory (plos_connector)

Reply:
Updated name to plos_main for clarity